cmake/Modules/LocalAvm.cmake

```diff
@@ -0,0 +1,11 @@
+# Building the avm repository generates files such as "libaom.a" because it is a fork of aom.
+set(LIB_FILENAME "${AVIF_SOURCE_DIR}/ext/avm/build.libavif/${CMAKE_STATIC_LIBRARY_PREFIX}aom${CMAKE_STATIC_LIBRARY_SUFFIX}")
```

Comment:
That means we do not support local builds of shared libraries? But that will be fixed with ExternalProject() right?

Reply:
This isn't actually a change from the current CMakeLists.txt:

https://github.com/AOMediaCodec/libavif/blob/9dae38279b7da440aa443df6c09a2e2079b39c64/CMakeLists.txt#L646-L648

It was already using the static suffix. The fact that it wasn't using the static prefix didn't really matter, because the shared and static prefixes are the same on a given platform, but it wasn't quite right. The build scripts all currently have `-DBUILD_SHARED_LIBS=OFF`, and I imagine the FetchContent and ExternalProject builds will likewise override BUILD_SHARED_LIBS before calling `add_subdirectory`. If the local dependencies were shared libraries they would need to be loaded at runtime, and so you would actually need to install them on the system to use with libavif.